src/abstractdataframe/abstractdataframe.jl

```diff
@@ -816,7 +817,8 @@ function Base.convert(::Type{Matrix{T}}, df::AbstractDataFrame) where T
         catch err
             if err isa MethodError && err.f == convert &&
                !(T >: Missing) && any(ismissing, col)
-                error("cannot convert a DataFrame containing missing values to Matrix{$T} (found for column $name)")
+                error("cannot convert a DataFrame containing missing values to Matrix{$T} " *
```

Comment:
maybe change this to `ArgumentError`?

Reply:
@bkamins I agree with you. With this PR, I just focused on line lengths. As you advised, I will commit the change.